xbmc/settings/Settings.h

```diff
@@ -82,6 +82,7 @@ class CSettings : public CSettingsBase, public CSettingCreator, public CSettingC
   static const std::string SETTING_VIDEOLIBRARY_SHOWEMPTYTVSHOWS;
   static const std::string SETTING_LOCALE_AUDIOLANGUAGE;
   static const std::string SETTING_VIDEOPLAYER_PREFERDEFAULTFLAG;
+  static const std::string SETTING_VIDEOPLAYER_DONTCHECKEXTERNALAUDIONAMES;
```

Comment:
There is no need to define this setting here. It is only used in Util.cpp so that's the file where it should be defined.

Reply:
Got it. I moved it into function since nothing else is using it.